go.mod

```diff
@@ -1,13 +1,17 @@
 module github.com/jzelinskie/cobrautil/v2
 
-go 1.18
+go 1.21
```

Comment:
Do we need to bump this? In libraries, it's actually beneficial to keep this as low as possible for compatibility reasons.

Reply:
Nope, that happened as a result of `go mod tidy`, i think.

Reply:
@jzelinskie I asked @ecordell about this and he said that his understanding is that `go mod tidy` will only bump this if one of the added deps requests a higher go version. I think this is probably a result of pulling in new deps rather than anything that I intentionally bumped. Is that okay?

Reply:
That makes sense! Let's let it bump it.